src/Infrastructure/LeanCode.ForceUpdate/LeanCode.ForceUpdate.Services/CQRS/VersionSupportQH.cs

```diff
@@ -0,0 +1,57 @@
+using LeanCode.CQRS.Execution;
+using LeanCode.ForceUpdate.Contracts;
+using Microsoft.AspNetCore.Http;
+using VersionSupport = LeanCode.ForceUpdate.Contracts.VersionSupport;
+
+namespace LeanCode.ForceUpdate.Services.CQRS;
+
+public class VersionSupportQH : IQueryHandler<VersionSupport, VersionSupportDTO?>
+{
+    private readonly IOSVersionsConfiguration iOSConfiguration;
+    private readonly AndroidVersionsConfiguration androidConfiguration;
+    private readonly VersionHandler versionHandler;
+
+    public VersionSupportQH(
+        IOSVersionsConfiguration iOSConfiguration,
+        AndroidVersionsConfiguration androidConfiguration,
+        VersionHandler versionHandler
+    )
+    {
+        this.iOSConfiguration = iOSConfiguration;
+        this.androidConfiguration = androidConfiguration;
+        this.versionHandler = versionHandler;
+    }
+
+    public Task<VersionSupportDTO?> ExecuteAsync(HttpContext context, VersionSupport query)
+    {
+        if (!Version.TryParse(query.Version, out var version) || !Enum.IsDefined(typeof(PlatformDTO), query.Platform))
+        {
+            return Task.FromResult<VersionSupportDTO?>(null);
+        }
+
+        return query.Platform switch
+        {
+            PlatformDTO.IOS
+                => Task.FromResult<VersionSupportDTO?>(
+                    (VersionSupportDTO)
+                        versionHandler.CheckVersion(
+                            version,
+                            iOSConfiguration.MinimumRequiredVersion,
+                            iOSConfiguration.CurrentlySupportedVersion,
+                            context
+                        )
+                ),
+            PlatformDTO.Android
+                => Task.FromResult<VersionSupportDTO?>(
+                    (VersionSupportDTO)
+                        versionHandler.CheckVersion(
+                            version,
+                            iOSConfiguration.MinimumRequiredVersion,
+                            iOSConfiguration.CurrentlySupportedVersion,
```

Comment:
Shouldn't this use `androidConfiguration`?

```suggestion
                            androidConfiguration.MinimumRequiredVersion,
                            androidConfiguration.CurrentlySupportedVersion,
```